src/System.IO.FileSystem/tests/Directory/Directory_EnumerableAPIs.cs

```diff
@@ -3,742 +3,234 @@
 
 using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
 using System.IO;
-using System.Security;
+using System.Linq;
 using Xunit;
 
 namespace EnumerableTests
 {
-    public class Directory_EnumerableTests
+    public class Directory_EnumerableTests : IClassFixture<TestFileSystemEntries>
```

Comment:
@FiveTimesTheFun - FYI our internal test infrastructure doesn't support IClassFixture so we will either need to add support for that or do this in a different way when run this test there.